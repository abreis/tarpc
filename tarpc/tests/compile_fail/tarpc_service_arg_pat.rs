#![allow(incomplete_features)]
#![feature(generic_associated_types)]

#[tarpc::service]
trait World {
    async fn pat((a, b): (u8, u32));
}

fn main() {}
