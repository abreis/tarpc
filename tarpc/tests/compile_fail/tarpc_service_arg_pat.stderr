error: patterns aren't allowed in RPC args
 --> $DIR/tarpc_service_arg_pat.rs:6:18
  |
6 |     async fn pat((a, b): (u8, u32));
  |                  ^^^^^^
